community/membership.md

```diff
@@ -0,0 +1,286 @@
+
+### Responsibilities and privileges
+
+The following apply to the subproject for which one would be an lead.
+
+- Make and approve technical design decisions for the subproject.
+- Set technical direction and priorities for the subproject.
+- Define milestones and releases.
+  - Decides on when PRs are merged to control the release scope.
+- Mentor and guide approvers, reviewers, and contributors to the subproject.
+- Escalate reviewer and maintainer concerns (i.e. responsiveness, availability, and general contributor community health) to the TC.
+- Ensure continued health of subproject
+  - Adequate test coverage to confidently release
+  - Tests are passing reliably (i.e. not flaky) and are fixed when they fail
+- Ensure a healthy process for discussion and decision making is in place.
+- Work with other subproject leads to maintain the project's overall health and success holistically
+
+
+
+## Inactive members
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of one of the Argoproj GitHub organizations comes with
+an [elevated set of permissions]. These capabilities should not be used by those
+that are not familiar with the current state of the Argo project.
+
+Therefore members with an extended period away from the project with no activity
+will be removed from the Argoproj Github Organizations and will be required to
+go through the org membership process again after re-familiarizing themselves
+with the current state.
+
+
+### How inactivity is measured
+
+Inactive members are defined as members of one of the Argoproj Organizations
+with **no** contributions across any organization within 12 months. This is
+measured by the CNCF [DevStats project].
+
+**Note:** Devstats does not take into account non-code contributions. If a
+non-code contributing member is accidentally removed this way, they may open an
+issue to quickly be re-instated. 
+
+After an extended period away from the project with no activity
+those members would need to re-familiarize themselves with the current state
+before being able to contribute effectively. 
+
+
+## Argoproj Ecosystem
+
+There are related [Argoproj GitHub organizations], such as [argoproj-labs].
+If you are a Argoproj org member, you are implicitly eligible
+for membership in related orgs, and can request membership when it becomes
+relevant, by creating a PR directly or [opening an issue][membership request]
+against the argoproj/argoproj repo, as above.
+
+However, if you are a member of any of the related
+[Argoproj GitHub organizations] but not of the [Argoproj org],
+you will need explicit sponsorship for your membership request.
+
+### Actively used GitHub Organizations
```

Comment:
This should be "Argoproj GitHub organizations" instead?

Reply:
These are "github organizations", of which we have currently two (argoproj and argoproj-labs) so I think "github orgs" is appropriate title.